app/view_objects/assessor_interface/assessment_section_view_object.rb

```diff
@@ -63,6 +63,10 @@ def online_checker_url
           region.country.teaching_authority_online_checker_url
     end
 
+    def professional_standing?
```

Comment:
I realise it's a super simple method, but since it's not private I think we should write a couple of tests for this.